### DISCUSSION TITLE
Create a homebrew formula

### DISCUSSION THREAD

**Discussion by martin-braun** [rustdesk/rustdesk#3984] [General]
Could you please consider adding a [Homebrew formula](https://docs.brew.sh/Formula-Cookbook) for the release?

Thank you.

**Comment by dylanmtaylor** [comment]
I bet CI could be added to automatically build this using GitHub actions. Mac runners are provided by MS and could be used to generate a dmg file

**Comment by rustdesk** [comment]
> CI

Wondering if someone can add this CI? including signing dmg.
